Completion.py

```diff
@@ -78,13 +111,23 @@ def get_file_path(filepath=None):
         try:
             filepath = filepath.replace(from_prefix, to_prefix)
         except:
-            sublime.status_message(GET_PATH_ERROR_MSG.format(from_prefix, to_prefix))
+            sublime.status_message(
+                GET_PATH_ERROR_MSG.format(from_prefix, to_prefix))
     return filepath
 
 
 def notify_func(filepath, content, callback):
     settings = read_settings()
-    cli = http_client.YcmdClient(settings["server"], settings["port"], settings["hmac"])
+    if (settings['use_auto']):
+        server = 'http://127.0.0.1'
```

Comment:
If machine has only ipv6 interfaces, it would not work.
I suggest server = 'http://localhost